File: TOMIMSV4/Entity/Transacciones/PreFactura/Trans_Prefactura_Erp/clsBeTrans_Prefactura_Erp.vb
Public Class clsBeTrans_prefactura_erp
    Implements ICloneable

    Public Property IdPrefacturaEnc As Integer
    Public Property Nit As String
    Public Property IdCliente_facturar As Integer
    Public Property Codigo_acuerdo As Integer
    Public Property IdCliente As Integer
    Public Property Moneda As String
    Public Property Periodo As String
    Public Property Mercaderia As String
    Public Property TipoCambio As Double
    Public Property Observaciones As String
    Public Property Detalle As List(Of clsBeTrans_prefactura_erp_det)

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Prefactura/clsBeTrans_prefactura_erp.cs
namespace WMS.EntityCore.Prefactura
{
    public class clsBeTrans_prefactura_erp : ICloneable
    {
        public int IdPrefacturaEnc { get; set; }
        public string Nit { get; set; } = "";
        public int IdCliente_facturar { get; set; }
        public int Codigo_acuerdo { get; set; }
        public int IdCliente { get; set; }
        public string Moneda { get; set; } = "";
        public string Periodo { get; set; } = "";
        public string Mercaderia { get; set; } = "";
        public double TipoCambio { get; set; }
        public string Observaciones { get; set; } = "";
        public List<clsBeTrans_prefactura_erp_det> Detalle { get; set; } = new List<clsBeTrans_prefactura_erp_det>();

        public clsBeTrans_prefactura_erp()
        {
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}